src/features/GetCommands.ts

```diff
@@ -0,0 +1,139 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+// tslint:disable:no-console
```

Comment:
I don't think we need this anymore

Reply:
Assume you mean the tslint. I agree! This was never meant to make it into a commit, but apparently it did. Removed now.